api/health/service.md

```diff
@@ -0,0 +1,214 @@
+---
+tags: [AvalancheGo APIs]
+description: This page is an overview of the Health API associated with AvalancheGo. This API can be used for measuring node health.
+sidebar_label: Health API
+pagination_label: Health API
+---
+
+# Health API
+
+This API can be used for measuring node health.
+
+:::info
+
+This API set is for a specific node, it is unavailable on the [public server](/tooling/rpc-providers.md).
+
+:::
```

Comment:
This is how docusaurus formats [admonitions](https://docusaurus.io/docs/markdown-features/admonitions). 
<img width="1030" alt="image" src="https://github.com/ava-labs/avalanchego/assets/99047250/62a28676-2f6a-4b6a-8bec-c2eb2db4e357">
It will unfortunately not look as pretty when viewed on gh, but it important for call outs on the docs site.